umbral/keys.py

```diff
@@ -239,3 +261,61 @@ def __eq__(self, other):
         else:
             is_eq = False
         return is_eq
+
+
+class UmbralKeyingMaterial(AbstractUmbralKeyingMaterial):
+    """
+    This class handles keying material for Umbral, by allowing deterministic
+    derivation of UmbralPrivateKeys based on labels. 
+    Don't use this key material directly as a key.
+    
+    """
+
+    def __init__(self, keying_material: bytes=None):
+        """
+        Initializes an UmbralKeyingMaterial.
+        """
+        if keying_material:
+            if len(keying_material) < 32:
+                raise ValueError("UmbralKeyingMaterial must have size at least 32 bytes.")
+            self.keying_material = keying_material
+        else:
+            self.keying_material = os.urandom(32)
```

Comment:
I believe that Blake2b can fit 64 bytes of entropy in it. Perhaps we generate 64 bytes from urandom and utilize all the available entropy.

Reply:
OK